tools/klee-stats/klee-stats

```diff
@@ -45,6 +45,9 @@ Legend = [
     ('Tcex', 'time spent in the counterexample caching code'),
     ('Tfork', 'time spent forking'),
     ('TResolve', 'time spent in object resolution'),
+    ('QCexCMisses', 'Counter example cache misses'),
+    ('QCexCHits', 'Counter example cache hits'),
+    ('QCex', 'Counter example cache')
```

Comment:
What is this statistic?  Is it the size of the CEX cache?  The name "Counter example cache" is not clear.